remove defaulting to 'primary' election after user uses geolocation

valfaro123 opened the pull request.

jessicaonly commented:
Hey @valfaro123 -- thanks so much for your contribution! 

This works well! The only update that I might suggest is tinkering around with the routing -- now, if you use the geolocation, it redirects you to `/null/[state]` until you pick `Primary` or `General`.  Maybe we could redirect people to `/state/[state]` or even just `/[state]` until they pick primary or general? (I think you could make this update around lines 125 and 131 of the `<Suggestion />` component, here!
https://github.com/dedline-io/dedline-io/blob/92214471f08a478ba66444f7a11c2d5a51bddf94/src/Components/Suggestion/index.jsx#L122-L131

valfaro123 commented:
Hello @jessicaonly 
I went ahead and changed the redirect at line 57 in the suggestion component to check for a null primaryOrGeneral and set the navigation accordingly. Alternatively, I found setting the initial state of  primaryOrGeneral to an empty string also fixed the issue. Thank you for making my first contribution to open source an awesome experience!

jessicaonly commented:
> Hello @jessicaonly I went ahead and changed the redirect at line 57 in the suggestion component to check for a null primaryOrGeneral and set the navigation accordingly. Alternatively, I found setting the initial state of primaryOrGeneral to an empty string also fixed the issue. Thank you for making my first contribution to open source an awesome experience!

Looks great! Thank you so much for contributing! ☺️
